manual/browserstack-testing.md

```diff
@@ -0,0 +1,18 @@
+# BrowserStack Testing
+
+## Testing locally running or TDR integration environment
+
+BrowserStack local testing allows the testing of non-public applications
```

Comment:
Can you add a line like: "We have a National Archives organisation account for BrowserStack. Ask the Digital Services frontend developers to set up an account for you within the organisation." Just in case new devs think they have to create their own account.

Reply:
Added